Баженова Анна

bazhenova opened the pull request.

bazhenova committed:
Решение задачи

maxvipon commented on test/flow-test.js:
```diff
@@ -0,0 +1,181 @@
+            assert.ok(spy.callCount === 3);
+        });
+
+        it('should call callback with first error if one of function failed', function () {
```
И какая же функция в тесте `failed`?

bazhenova replied:
Да, согласна. Вообще ничего не понятно из названия.

maxvipon commented on test/flow-test.js:
```diff
@@ -0,0 +1,181 @@
+            assert.ok(spy.calledAfter(spy3));
+        });
+
+        it('should call callback with first error if one of function failed', function () {
```
А почему не проверяешь ошибку в первой функции по аналогии с [тестом для `Serial`](https://github.com/urfu-2015/webdev-tasks-3/pull/11/files#diff-2bfd95d2a1d5a837fca569fbd62b3725R33)?

bazhenova replied:
Это вообще был неправильный тест :D

bazhenova committed:
Правка тестов

1) исправлены глобальные косяки в тестах
2) переименованы некоторые тесты
3) добавлены тесты

bazhenova commented:
@maxvipon Постаралась все поправить.

maxvipon commented:
:ok:
